src/olympia/conf/dev/settings.py

```diff
@@ -47,8 +47,8 @@
 SITE_URL = 'https://' + DOMAIN
 SERVICES_URL = env('SERVICES_URL',
                    default='https://services.addons-dev.allizom.org')
-STATIC_URL = 'https://addons-dev-cdn.allizom.org/static/'
-MEDIA_URL = 'https://addons-dev-cdn.allizom.org/user-media/'
+STATIC_URL = '%s/static/' % CDN_HOST
+MEDIA_URL = '%s/user-media/' % CDN_HOST
```

Comment:
I kinda hate `urljoin` because this fails silently:
`urljoin('https://cdn.example.com/', '/lol', 'ooops')`

(`urlparse.urljoin` does not work like `os.path.join` does and only joins the first 2 arguments)